pkg/platform/platform.go

```diff
@@ -19,6 +19,15 @@ const ( // image format enum
 	FORMAT_OVA
 )
 
+type Bootloader int
+
+const ( // bootloader enum
```

Comment:
I *love* this moving of bootloaders to platforms instead of toggling on architecture in the `os` pipeline so feel free to ignore this, but there are multiple stages of bootloaders and it seems like we're conflating them into a single bootloader enum?

Reply:
> Feel free to ignore this, but there are multiple stages of bootloaders and it seems like we're conflating them into a single bootloader enum?

I don't want to ignore this, but maybe not something to address in this PR.  I would love to have a more _correct_ model of reality in the way the code and options are structured.  We should definitely come back to this.